src/current/_includes/common/basic-terms.md

```diff
@@ -34,24 +34,28 @@ Service accounts operate under a unified authorization model with organization u
 
-To learn more, refer to [Manage Service Accounts]({{ link_prefix }})managing-access.html#manage-service-accounts)
+To learn more, refer to [Manage Service Accounts]({{ link_prefix }}managing-access.html#manage-service-accounts).
 
 #### CockroachDB {{ site.data.products.basic }} cluster
 
-A CockroachDB {{ site.data.products.cloud }} cluster deployed on request for a specific customer in *shared* network and compute infrastructure.
+A CockroachDB {{ site.data.products.cloud }} cluster with minimal operational features deployed in *shared* network and compute infrastructure.
 
-#### CockroachDB {{ site.data.products.dedicated }} cluster
+#### CockroachDB {{ site.data.products.standard }} cluster
 
-A CockroachDB {{ site.data.products.cloud }} cluster deployed on request for a specific customer, in a cloud provider's network and compute infrastructure *dedicated* to that customer, which can be distributed over multiple regions for added disaster-resilience. In addition to infrastructure isolation, dedicated clusters can be customized with advanced network, identity-management, and encryption-related security features.
+A CockroachDB {{ site.data.products.cloud }} cluster with full operational features and provisioned capacity, deployed in *shared* network and compute infrastructure.
+
+#### CockroachDB {{ site.data.products.advanced }} cluster
+
+A CockroachDB {{ site.data.products.cloud }} cluster with full operational capacity deployed in a cloud provider's network and compute infrastructure *dedicated* to each customer, which can be distributed over multiple regions for added disaster-resilience. In addition to infrastructure isolation, Advanced clusters can be customized with advanced security features for PCI DSS and HIPAA [compliance]({{ link_prefix }}compliance.html) at an additional cost.
 
 #### Request Unit (RU)
 
-Represents the compute and I/O resources used by a query. All database operations in CockroachDB {{ site.data.products.basic }} and {{ site.data.products.standard }} cost a certain amount of RUs depending on the resources used. For example, a "small read" might cost 2 RUs, and a "large read" such as a full table scan with indexes could cost a large number of RUs. You can see how many Request Units your cluster has used on the [Cluster Overview]({{ link_prefix }}cluster-overview-page.html#request-units) page.
+{% include cockroachcloud/request-units.md %}
 
 #### Resource limits
 
 The maximum amounts of storage and RUs a CockroachDB {{ site.data.products.basic }} and {{ site.data.products.standard }} cluster can use in a particular billing period. The amount you are billed is based on the actual resources the cluster used during that billing period.
```

Comment:
It would be helpful to compare RU with a unit customers are already familiar with - vCPU.
Each 500 RUs/sec equals approximately 1 vCPU.  
cc @marksoper to see if he wants to add this comparison in docs

Reply:
We do have similar mapping in the [Understand your usage](https://www.cockroachlabs.com/docs/cockroachcloud/serverless-resource-usage#understand-resource-consumption-in-cockroachdb-serverless) doc. Let me know if we want to add the mapping to the glossary (the current doc in this PR)

Reply:
Yes. Let's add a reference

Reply:
Done.